services/libssh/cve_2018_10933_service.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+"""LIBSSH CVE-2018-10933 Honeycomb Service."""
+from __future__ import unicode_literals
+
+from base_service import ServerCustomService
+
+from cve_2018_10933_server import SSHServer
+from consts import CVE_ALERT_TYPE, EVENT_TYPE_FIELD_NAME, ORIGINATING_IP_FIELD_NAME, ORIGINATING_PORT_FIELD_NAME,\
+    SOCK_IP_POSITION, SOCK_PORT_POSITION, CVE_PORT_FIELD
+
+
+class CVEService(ServerCustomService):
+    """LIBSSH CVE-2018-10933 Honeycomb Service."""
+
+    def __init__(self, *args, **kwargs):
+        super(CVEService, self).__init__(*args, **kwargs)
+        self.server = None
+        self.transport = None
+        self.chan = None
+
+    def alert(self, sock, event_type=CVE_ALERT_TYPE, *args, **kwargs):
+        """Send alert."""
+        params = {
+            EVENT_TYPE_FIELD_NAME: event_type,
+            ORIGINATING_IP_FIELD_NAME: sock.getpeername()[SOCK_IP_POSITION],
```

Comment:
Adding constants for 1 and 0 here is a bit over the top IMO.
You can also just do tuple unpacking e.g.
`ip, port = sock.getpeername()`
`params = ...`